internal/services/azure_reservation_service.go

```diff
@@ -77,14 +77,15 @@ func CreateAzureReservation(w http.ResponseWriter, r *http.Request) {
 
 	var azureImageName string
 	if strings.HasPrefix(payload.ImageID, "composer-api") {
```

Comment:
if you want the Marketplace images to work, just adjust this statement, here (the first branch of this if) is where we are handling the direct Azure images, we just expected the name to be from IB, which names the images with this prefix :)